docs/src/examples/components/Popup/Types/PopupCustomTargetExample.shorthand.tsx

```diff
@@ -16,7 +16,11 @@ class PopupExample extends React.Component {
         {/* CUSTOM DOM ELEMENT is used as target for Popup */}
         <Popup
           target={this.state.popupTarget}
-          trigger={<Icon name="question" circular bordered styles={{ cursor: 'pointer' }} />}
+          trigger={
+            <div>
```

Comment:
just to be sure - could you, please, suggest why `<div>` is necessary here? Is it because of some layout concerns?

Reply:
Yes, it's layout. I made this change because of a failing screen reader test.

Reply:
However, after pulling in the latest looks like the trigger was changed to a button which doesn't require a wrapper div